x-pack/plugins/threat_intelligence/public/modules/query_bar/components/filter_out/filter_out.tsx

```diff
@@ -102,9 +58,59 @@ export const FilterOut: VFC<FilterOutProps> = ({ data, field, type, as: Componen
         iconSize="s"
         size="xs"
         color="primary"
-        onClick={filterOut}
+        onClick={filterFn}
         {...props}
       />
     </EuiToolTip>
   );
 };
+
+/**
+ * Retrieves the indicator's field and value, then creates a new {@link Filter} and adds it to the {@link FilterManager}.
+ *
+ * This component is to be used in an EuiContextMenu.
+ *
+ * @returns filter in item for a context menu
+ */
+export const FilterOutContextMenu: VFC<FilterOutProps> = ({ data, field, ...props }) => {
+  const { filterFn } = useFilterInOut({ indicator: data, field, filterType: FilterOut });
+  if (!filterFn) {
+    return <></>;
+  }
+
+  return (
+    <EuiContextMenuItem key="filterOut" icon="minusInCircle" size="s" onClick={filterFn} {...props}>
```

Comment:
props spreading not the best pattern, https://vhudyma-blog.eu/react-antipatterns-props-spreading/ is it possible to avoid it?

Reply:
yup I removed all the `...props` instances in the plugin